fizzbuzztree/fizzbuzztree.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+
+const fizzBuzzTree = (tree) => {
+  let fizzBuzzOutput = '';
+  const inOrderTraversal = (rootNode) => {
+    if (!rootNode) {
+      return undefined;
+    }
+    inOrderTraversal(rootNode.left);
+    if (rootNode.value % 3 === 0 && rootNode.value % 5 === 0) {
```

Comment:
This module looks good, one trick to fizzbuzz is that you can also make this first condition like so:

```
if (rootNode.value % 15 === 0) {
  // do stuff here
}
```